spec/providers/coreIntegration/tcpsocket.integration.src.js

```diff
@@ -141,5 +174,90 @@ module.exports = function (provider, setup) {
     });
   });
 
-  // TODO: add tests for tcpsocket.secure, accepting multiple.
+  it("Secures a socket", function (done) {
+    socket.connect('www.google.com', 443, function () {
+      var prepared;
+      // prepareSecure is Chrome specific, see freedom-for-chrome
+      socket.prepareSecure(function () {
+        prepared = true;
+      });
+      // TODO: timeout approach sucks, preferred approach:
```

Comment:
Added a bit more detail to the wiki (referenced in that file): https://github.com/freedomjs/freedom/wiki/prepareSecure-API-Usage